cfy_manager/components/postgresql_server/postgresql_server.py

```diff
@@ -675,14 +710,14 @@ def _determine_cluster_status(self, db_nodes):
                 )
             )
             status = max(status, self.DOWN)
-        if etcd_followers < 1:
+        if etcd_followers < majority_requirement:
             logger.error(
                 'No etcd followers found, cluster consensus lost.'
```

Comment:
that is no longer true

Reply:
In what situation is it not true?

Reply:
If you're talking about the 1 node case, that'll result in majority_requirement = 1 // 2, which is 0.

Reply:
OK, a quick out-of-band discussion clarified: The message itself was wrong. Updated.